fixtures/prov_filter.py

```diff
@@ -1,5 +1,15 @@
+import re
 from utils.conf import cfme_data
 from utils.log import logger
+from utils.version import current_version
+
+
+mapping = {'<': lambda o, v: o.__cmp__(v) < 0,
+           '<=': lambda o, v: o.__cmp__(v) < 0 or o.__cmp__(v) == 0,
+           '==': lambda o, v: o.__cmp__(v) == 0,
+           '!=': lambda o, v: o.__cmp__(v) != 0,
+           '>': lambda o, v: o.__cmp__(v) > 0,
+           '>=': lambda o, v: o.__cmp__(v) > 0 or o.__cmp__(v) == 0}
```

Comment:
Also, since versions support `__cmp__`, the lambda should be able to do the op:

`{'<=': lambda o, v: o <= v}`